### PR TITLE
add faq for sl error

### PR DESCRIPTION
**Pull request by mirnawong1**
adding an SL faq to help users unblock themselves when they encounter an alpn error per [slack thread](https://dbt-labs.slack.com/archives/C05K4R7KZ5Z/p1700674655839969?thread_ts=1700032248.113419&cid=C05K4R7KZ5Z)

### CODE REVIEW

File: website/docs/faqs/Troubleshooting/sl-alpn-error.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: I'm receiving an `Failed ALPN` error when trying to connect to the dbt Semantic Layer.
+description: "To resolve the 'Failed ALPN' error in the dbt Semantic Layer, create a SSL interception exception for the dbt Cloud domain."
+sidebar_label: 'Use SSL exception to resolve `Failed ALPN` error'
+---
+
+If you're receiving a `Failed ALPN` error when trying to connect the dbt Semantic Layer with the available [data integration tool](/docs/use-dbt-semantic-layer/avail-sl-integrations), it typically happens when connecting from a computer behind a corporate VPN or Proxy, such as Zscaler. 
+
+To fix this, add an SSL interception exception for the dbt Cloud domain in your settings.
```
Comment (aiguofer) [inline website/docs/faqs/Troubleshooting/sl-alpn-error.md]: I think adding a bit of context on why the error occurs might help users work with their IT department to figure out. Maybe something like:

>The problem is due to the proxy interfering with the TLS handshake. The Semantic Layer uses gRPC/HTTP2 to connect, so if your proxy does not support this you'll need to add an exception for the dbt Cloud domain. If your proxy does support this, you'll need to make sure it's configured correctly to allow ALPN, or create an exception for the dbt Cloud domain.

File: website/docs/faqs/Troubleshooting/sl-alpn-error.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: I'm receiving an `Failed ALPN` error when trying to connect to the dbt Semantic Layer.
+description: "To resolve the 'Failed ALPN' error in the dbt Semantic Layer, create a SSL interception exception for the dbt Cloud domain."
+sidebar_label: 'Use SSL exception to resolve `Failed ALPN` error'
+---
+
+If you're receiving a `Failed ALPN` error when trying to connect the dbt Semantic Layer with the available [data integration tool](/docs/use-dbt-semantic-layer/avail-sl-integrations), it typically happens when connecting from a computer behind a corporate VPN or Proxy, such as Zscaler.
```
Comment (aiguofer) [inline website/docs/faqs/Troubleshooting/sl-alpn-error.md]: another product we experienced issues with was Check Point. Might be worth changing to `such as Zscaler or Check Point`

**Comment by mirnawong1** [comment]
wonderful, thank you so much Diego! i've folded in your changes and tweaked it further -- can you double check it please? thank you!

**Review by aiguofer — APPROVED** [review]
LGTM! Thanks so much :)

**Pull request merged**